### PR TITLE
Version 2.1.4

### PR DESCRIPTION
**Pull request by timothysmith0609**
Version 2.1.4 release

### CODE REVIEW

**Review by karanthukral — APPROVED** [review]
🙇‍♂️

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,12 @@
 ## next
 
+## 2.1.4
+
 *Other*
 - Dropped support for Ruby 2.5 due to EoL. [#782](https://github.com/Shopify/krane/pull/782).
 - Attempt to batch run server-side apply in validation phase instead of dry-running each resource individually [#781](https://github.com/Shopify/krane/pull/781).
+- Add k8s 1.19 and 1.20 to CI [#777](https://github.com/Shopify/krane/pull/777).
```
Comment (dturn) [inline CHANGELOG.md:8]: I don't think we normally list these.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,8 +1,12 @@
 ## next
 
+## 2.1.4
+
 *Other*
```
Comment (dturn) [inline CHANGELOG.md:5]: Should we just drop the other header? If not, 781 & 765 look more like `Enhancements` than other to me.

**Review by dturn — APPROVED** [review]
I think we need to call out https://github.com/Shopify/krane/pull/779 somewhere.

**Pull request merged**